lib/elixir_boilerplate/schema.ex

```diff
@@ -0,0 +1,15 @@
+defmodule ElixirBoilerplate.Schema do
+  defmacro __using__(_) do
+    quote do
+      use Ecto.Schema
+
+      import Ecto.Changeset
+
+      alias Ecto.{Schema, UUID}
+
+      @primary_key {:id, :binary_id, autogenerate: true}
+      @foreign_key_type :binary_id
+      @timestamps_opts [type: :utc_datetime_usec]
```

Comment:
We also have `migration_timestamps: [type: :utc_datetime_usec]` in the config for a lot of our app.